docs/zh_cn/user_guides/visualization_feature_map.md

```diff
@@ -0,0 +1,201 @@
+# wandb记录特征图可视化
+
+MMSegmentation 1.x 提供了Weights & Biases的后端支持，方便对项目代码结果的可视化和管理。
```

Comment:
```suggestion
MMSegmentation 1.x 提供了 Weights & Biases 的后端支持，方便对项目代码结果的可视化和管理。
```

Reply:
We prefer to leave a whitespace between Chinese and English characters or arabic numerals at default.